paper/src/main/java/com/programmerdan/minecraft/simpleadminhacks/hacks/basic/NoGraceWIthDepth.java

```diff
@@ -10,8 +10,8 @@
 import org.bukkit.inventory.ItemStack;
 import org.bukkit.potion.PotionEffectType;
 
-public final class NoGraceWIthDepth extends BasicHack {
-	public NoGraceWIthDepth(final SimpleAdminHacks plugin, final BasicHackConfig config) {
+public final class NoGraceWithDepth extends BasicHack {
```

Comment:
Well done for figuring out how to modify an existing pull request :) As for this fix, you'll also need to modify the file-name itself to fix the typo too. In Java, classes must ALWAYS be named the same as the actual file.